catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -166,27 +223,44 @@ private void writeRemoteUpdates(List<Metacard> remoteMetacardsToUpdate)
                 return null;
             });
         } catch (PrivilegedActionException e) {
-            String message = "Error writing remote updates.";
-            LOGGER.debug("{} Metacard IDs: {}", message, remoteMetacardsToUpdate);
-            throw new FederationAdminException(message, e);
+            throw new FederationAdminException("Error writing remote updates", e);
```

Comment:
Was just making this consistent with the other CUD operations but I see how the mcard ids could be useful so I put it back in.